pkg/datasource.go

```diff
@@ -142,8 +142,13 @@ func (plugin *GrafanaAzureDXDatasource) QueryData(ctx context.Context, req *back
 		return nil, err
 	}
 
+	email := "Noninteractive"
```

Comment:
should be userLogin now

Reply:
Actually I can fix that in the opt in pr. I'll merge. Again thanks for contributing!

Reply:
Thank you !!! Yes, I totally missed renaming the variable -- would have been very confusing.